source/helpers/math.ts

```diff
@@ -0,0 +1,31 @@
+export function lerp(min: number, max: number, t: number): number {
+	return min + ((max - min) * t);
+}
+
+export function invlerp(min: number, max: number, x: number): number {
+	return (x - min) / (max - min);
+}
+
+export function calculateMinMax(values: number[]): {min: number; max: number} {
+	return {
+		min: Math.min(...values),
+		max: Math.max(...values),
+	};
+}
+
+export function createHeatIndexFunc(values: number[]): (value: number) => number {
```

Comment:
Maybe `findHeatIndex`?

Reply:
It's a little weird since returns a function, not the head index directly

Reply:
@fregante do you have a better name for this function?

Reply:
I think it's a reasonable name, other than "Func" which can just be Function. I'm not sure why eslint didn't catch it https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prevent-abbreviations.md

Reply:
the "lerp" functions should be extended though. Use "linear interpolation" or something else. calculateMinMax can just be inlined as:

```js
const min = Math.min(...values);
const max = Math.max(...values);
```

Reply:
Made the changes in latest!